specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -20,14 +20,15 @@ options:
     namespace: "Azure.AI.OpenAI"
     clear-output-folder: true
     model-namespace: false
+    generate-protocol-methods: false
   "@azure-tools/typespec-java":
     package-dir: "azure-ai-openai"
     namespace: "com.azure.ai.openai"
     partial-update: true
     enable-sync-stack: true
     generate-tests: false
     custom-types-subpackage: "implementation.models"
-    custom-types: "FunctionCallPreset"
+    custom-types: "FunctionCallModelBase,FunctionCallPreset,FunctionCallPresetFunctionCallModel,FunctionNameFunctionCallModel"
```

Comment:
I think you need to merge `main`. There were recently removed due to the `access/usage` change.